embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/SessionSpanWriter.kt

```diff
@@ -28,8 +28,8 @@ internal interface SessionSpanWriter {
  * Represents a span event that can be added to the current session span.
  */
 internal class SpanEventData(
-    val name: String,
-    val timeNanos: Long,
+    val spanName: String,
+    val spanStartTimeNanos: Long,
```

Comment:
I must've missed this when I did the initial review - this is actually the span event time, not the start time of the span. so Maybe eventName and eventTimeNanos if we want to make this more clear?